src/govuk/core/_lists.scss

```diff
@@ -48,11 +48,23 @@
     }
   }
 
+  %govuk-list--spaced > li {
+    margin-bottom: govuk-spacing(3);
+
+    &:last-child {
```

Comment:
Does the original govuk-list remove the last child margin?

Reply:
The original list items don't have any bottom margins. But if they did then I think we would remove it from the last one.

Reply:
Hmm, they do on tablet and above:

https://github.com/alphagov/govuk-frontend/blob/5fcf41bc32df9f892f4df938994067c13433f041/src/govuk/core/_lists.scss#L42-L49

I think changing that behaviour might be a breaking change, but if we thought it made sense to bring them in line we could do that in 4.0?

Which raises another point – are we happy with the same spacing being applied on mobile?

Reply:
I would say let's figure out what the correct behaviour is and have it consistent, and do it in 4.0.0

Reply:
Looking at this again, the `ul` has a bottom margin of 15px (mobile) or 20px (tablet, desktop) and the bottom margin of the last `li` collapses into it. As far as I can tell this means that zero-ing out the last list item has no effect and so this can probably be removed to simplify things?

Reply:
Sounds good!